docs/100_producten/100_borgstelling/borgstelling-beeindigen.feature

```diff
@@ -1,14 +1,27 @@
 # language: nl
 Functionaliteit: Borgstelling beëindigen
 
-  Achtergrond: 
-    Gegeven het kenmerk "093e08db-6791-454b-a172-068099514907"
+  Achtergrond:
+    Gegeven een kredietbank
 
-  Scenario: Borgstelling beëindigen
+  Scenario: Borgstelling beëindigen van afgegeven aanvraag
     Gegeven een afgegeven borgstelling
+    En een beeindigdatum van "2022-01-01"
     Wanneer het "beëindiging borgstelling" bericht is verstuurd via het Schuldenknooppunt
-    En het Schuldenknooppunt bericht is verwerkt
-    Dan is de status van de borgstelling "beëindigd"
+    En het "beëindiging borgstelling" is verwerkt
+    Dan is de status van de borgstelling "BEEINDIGD"
     En is "de beëindiging" gearchiveerd
     En is de datum beëindiging juist geregistreerd
-    En is het "borgstelling beëindigd" bericht ontvangen door het Schuldenknooppunt
\ No newline at end of file
+    En is het "borgstelling beëindigd" bericht ontvangen door het Schuldenknooppunt
+
+  Scenario: Borgstelling beëindigen van actieve aanvraag
```

Comment:
Waarom is dit en scenario? Waarom is die niet hetzelfde als borgstelling beëindigen?

Reply:
Als een aanvraag in maatwerk staat, kan je ook een beeindiging bericht starten. Dan wordt de aanvraag beeindigd i.p.v. de borgstelling. Zie subprocess van het aanvraag borgstelling process.

Reply:
Ik zie het nu ook in het [proces](https://wsk.sbn.nl/main/producten/borgstelling/aanvragen-borgstelling/index.html#proces) staan. Ik was vergeten dat we dit expliciet hadden gemaakt.